backend/fittoring/src/main/java/fittoring/application/auth/service/AuthService.java

```diff
@@ -151,11 +171,7 @@ public MemberOauth registerOauthMember(OauthSignUpRequest request, String oauthS
                 Password.from(randomPw)
         );
         validateDuplicateLoginId(randomLoginId);
-        validateDuplicatePhone(request.phone());
-        memberRepository.save(member);
-        MemberOauth memberOauth = new MemberOauth(member, AuthProvider.KAKAO, oauthId);
-        memberOauthRepository.save(memberOauth);
-        return memberOauth;
+        return member;
```

Comment:
```suggestion
    private Member getRandomIdPwMember(OauthSignUpRequest request) {
        String randomLoginId = RandomStringUtils.randomAlphanumeric(20);
        validateDuplicateLoginId(randomLoginId);
        String randomPw = RandomStringUtils.randomAlphanumeric(20);
        return new Member(
                randomLoginId,
                request.gender(),
                request.name(),
                new Phone(request.phone()),
                Password.from(randomPw)
        );
    }

```
`validateDuplicateLoginId`의 순서를 바꾸면 코드가 조금 간결해질 것 같아요!

Reply:
반영했습니다~!